objectivec/GPBCodedInputStream.h

```diff
@@ -35,6 +35,34 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+CF_EXTERN_C_BEGIN
+
+/// GPBCodedInputStream exception name.
+extern NSString *const GPBCodedInputStreamException;
+
+/// NSError domain used for GPBCodedInputStream errors.
+extern NSString *const GPBCodedInputStreamErrorDomain;
+
+/// Error code for NSError with GPBCodedInputStreamErrorDomain.
+typedef NS_ENUM(NSInteger, GPBCodedInputStreamErrorCode) {
```

Comment:
Same, sorta seems like Message should be exposing a set and hiding the input stream as an impl detail.